From 1d444a409ce590c9ae4e800585bd36748fb99a10 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 10 May 2022 14:03:30 +0100
Subject: [PATCH] dtbindings: Fixup microchip,emc2305.yaml bindings

The bindings submitted to mainline had some issues, so fix them up.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../devicetree/bindings/hwmon/microchip,emc2305.yaml      | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/Documentation/devicetree/bindings/hwmon/microchip,emc2305.yaml
+++ b/Documentation/devicetree/bindings/hwmon/microchip,emc2305.yaml
@@ -5,13 +5,13 @@
 $id: http://devicetree.org/schemas/hwmon/emc2305.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Microchip EMC2305 RPM-based PWM Fan Speed Controller
+title: Microchip EMC230[1|2|3|5] RPM-based PWM Fan Speed Controller
 
 properties:
   compatible:
     enum:
-      - microcip,emc2305
-
+      - microchip,emc2305
+      - microchip,emc2301
   emc2305,pwm-min:
     description:
       Min pwm of emc2305
@@ -46,7 +46,7 @@ additionalProperties: false
 examples:
   - |
     fan {
-        emc2305,compatible = "microchip,emc2305";
+        compatible = "microchip,emc2305";
         emc2305,pwm-min = <0>;
         emc2305,pwm-max = <255>;
         emc2305,pwm-channel = <5>
